R/transformCounts.R

```diff
@@ -405,19 +399,22 @@ setMethod("relAbundanceCounts",signature = c(x = "SummarizedExperiment"),
     # Input check end
     
     # apply pseudocount, if it is numeric
-    if( is.numeric(pseudocount) ){
+    # in clr and alr, the pseudocount is handled in vegan::decostand
+    if( is.numeric(pseudocount) && !method %in% c("clr", "rclr", "alr") ){
```

Comment:
This could do but I see a potential risk in case vegan is now, or in later versions, using pseudocount for other methods than these three. But not sure what would be a better solution. 

If we use/need only vegan transformations here, then we could just port those and just pass the arguments to vegan. Perhaps it is worth checking, could simplify maintenance, if this would be an appropriate solution.

Reply:
Some of the transformations have a different name in vegan but at least most of the transformations we use are available there (as it is a generic ecology R pkg)

Reply:
All the transformation that we have are in vegan (log differs little bit)

Should we modify our function so that it is just a wrapper for vegan::decostand?

Reply:
I think it would make sense for now. This will simplify maintenance and reduce potential for error. It also makes quite a lot of sense to direct as much of the development of that part as possible to the same toolkit that is more generally used by ecologists.

We can have aliases for our favourite transformation names if we like.

The downside is if we want to add some non-vegan transformations later. Then it might be more work to change things agains. But this is not immediately in sight and perhaps it would not be that much work if there was something that has to be done that way.

Reply:
But: I have not evaluated this very thoroughlty, if you see reasons to not do this, let's discuss. Just that if there are no obvious major benefits, then I would just provide TreeSE interface to vegan here.

Reply:
This is good plan, I think. I will keep the framework for our own functions available since there will still be 2 transformations that are not calculated with vegan:

- pa (vegan pa does not have threshold option)
- log10 (in vegan, log transformation is done little bit differently. I think we could have both "log10" and "log".)

Reply:
Yes this is good. We should avoid having names that are identical with vegan but have a different meaning. In this case "log" (and log10) should be fine I think. It might be good to have log2 as well if it is easy to add.